packages/fiori/src/NotificationList.ts

```diff
@@ -39,9 +39,26 @@ type NotificationItemCloseEventDetail = NotificationItemEventDetail;
  * The `ui5-notification-list` web component represents
```

Comment:
The recommended structure of a component (see [here](https://workzone.one.int.sap/site#workzone-viewAll?sap-app-origin-hint=&/wiki/show/gnY0lJJRpoHghUaznFjmYo)) also has other sections like Properties, Events, Methods, etc... Do we need these?

Reply:
They will be generated from the metadata.